builder/vmware/common/driver_fusion6.go

```diff
@@ -124,6 +124,25 @@ func (d *Fusion6Driver) Verify() error {
 	return compareVersions(matches[1], VMWARE_FUSION_VERSION, "Fusion Professional")
 }
 
+func (d *Fusion6Driver) ToolsIsoPath(k string) string {
+	// Fusion 13.x.x changes tools iso location
```

Comment:
If this changes with version 13, I wonder if this may signal that we should implement a `Fusion13Driver` for this?

Reply:
There's quite a bit of "version-specific" naming in the plugin. My current view is that some of this could / should be simplified and that releases should consider focusing on versions that are supported by VMware. For example, Fusion 12 (and earlier) is now end-of-support, so is Workstation 16 (and earlier), and ESX 6.7 (and earlier).

Reply:
@lbajolet-hashicorp - feel free to ping me over Slack Connect to discuss if needed. Wilken knows how to find me if you have any issues.

**Ryan Johnson**
Senior Staff Solutions Architect | Product Engineering @ VMware, Inc.